### PR TITLE
Add dialog box for updating metadata

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/dcp-ingest-central#36

### CODE REVIEW

File: client/src/app/metadata-details-dialog/metadata-details-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    const newContent = formData['value'];
+    this.metadata['content'] = newContent;
+    this.metadata['validationState'] = 'Draft';
+    this.ingestService.patch(selfLink, this.metadata)
+      .subscribe(response => {
+        console.log('successful update', response);
+        this.dialogRef.close();
+      }, err => {
+        console.error('error', err);
+        this.dialogRef.close();
```
Comment (rdgoite) [inline client/src/app/metadata-details-dialog/metadata-details-dialog.component.ts:89]: When a connection error occurs the dialog box will just close, and the issue won't reported to the user?

**Review by rdgoite — APPROVED** [review]
Minor comments that can be addressed later. Might also want to consider writing tests for services.

File: client/src/app/metadata-details-dialog/metadata-details-dialog.component.ts (hunk 1 above)
Comment (aaclan-ebi) [reply client/src/app/metadata-details-dialog/metadata-details-dialog.component.ts:89]: yeah i still need to fix displaying of success and error messages. I'll add it soon... maybe won't make it for demo this afternoon. I'll just deploy the feature image in dev for now for the demo

Comment (aaclan-ebi) [reply client/src/app/metadata-details-dialog/metadata-details-dialog.component.ts:101]: Added displaying of success and error message upon editing

**Pull request merged**